docs/general/builtins/aggregation.rst

```diff
@@ -825,6 +825,28 @@ Example::
     executions of the aggregate function on the same data produce slightly
     differing results.
 
+.. _aggregation-top_k:
+
+``top_k(column, limit)``
+------------------------
+
+The ``top_k`` aggregate function computes an approximation of the most frequent
+values and their frequencies of a specific column. The result is an array
+containing the most common values and their frequencies. The limit is an optional
+parameter which defines how many elements should be returned which defaults to
+8. Internally the ``top_k`` aggregate function use an implementation of the
+`Top-k Elements in Data Streams`_ algorithm.
+
+Example::
+
+    cr> SELECT top_k(x) FROM (values (1), (2), (2), (3), (3), (3), (3)) as t (x);
```

Comment:
```suggestion
    cr> select top_k(country, 4) from sys.summits;
```

I think an example like this would be easier to understand. Using numbers is kinda confusing as the frequencies are numbers too so it is not clear which is which.

Reply:
👍